Jenkinsfile

```diff
@@ -50,7 +50,7 @@ def dhfCypressE2ETests(String mlVersion, String type){
                     ./gradlew clean;
                     cd marklogic-data-hub-central/ui/e2e;
                     chmod +x setup.sh;
-                    ./setup.sh;
+                    ./setup.sh dhs=false mlHost=localhost;
```

Comment:
What does the "dhs" property do? I'm not aware of it.

Reply:
We made few changes to the script setup.sh which is expecting the value "dhs=true/fase" based on the type of environment we are testing on. In Jenkins pipeline we are running cypress e2e tests on a local environment, so setting the property "dhs=false"

Reply:
Setting up dhs=true will run the tasks "hubdeployasdeveloper" and dhs=false will run "mldeploy"